barretenberg/cpp/src/barretenberg/dsl/acir_format/ivc_recursion_constraint.test.cpp

```diff
@@ -36,7 +36,7 @@ class IvcRecursionConstraintTest : public ::testing::Test {
         Builder circuit{ ivc->goblin.op_queue };
         GoblinMockCircuits::add_some_ecc_op_gates(circuit);
         MockCircuits::add_arithmetic_gates(circuit);
-
+        AggregationObject::add_default_pairing_points_to_public_inputs(circuit);
```

Comment:
Is it possible to just do this defaulting in circuit finalization instead of in all these individual places? Would that cause issues somehow?